packaging/linux/run_keybase

```diff
@@ -20,6 +20,9 @@ fi
 if fusermount -uz /keybase &> /dev/null ; then
   echo Unmounting /keybase...
 fi
+if killall kbupdater &> /dev/null ; then
+  echo Shutting down updater...
+fi
```

Comment:
I'd like to keep the `fusermount -u` and the `killall kbfsfuse` commands next to each other. Maybe move this down one?